api/websocket/server/relay.go

```diff
@@ -105,16 +105,18 @@ func (ws *WsServer) sendInboundMessage(clientID string, inboundMsg *pb.InboundMe
 	return success
 }
 
-func (ws *WsServer) sendInboundRelayMessage(relayMessage *pb.Relay) {
+func (ws *WsServer) sendInboundRelayMessage(relayMessage *pb.Relay, checkShouldSign bool) {
```

Comment:
The name is a bit confusing. I would recommend using `shouldSign` (same as local variable)